pkg/reconciler/pipelinerun/affinity_assistant.go

```diff
@@ -190,8 +203,32 @@ func affinityAssistantStatefulSet(name string, pr *v1beta1.PipelineRun, claimNam
 				"memory": resource.MustParse("100Mi"),
 			},
 		},
+		VolumeMounts: mounts,
 	}}
 
+	// TODO(QuanZhang-William): when introducing `coscheduling` flag with values `coschedule-pipelineruns` or `isolate-pipelineruns
```

Comment:
I don't think this TODO is actually true-- isn't this what this PR is supposed to address? If you do keep it around, please link an issue here rather than your username.

Reply:
SGTM, removed